File: app-funcionario/src/app/models/folha-pagamento.ts
import { Funcionario } from "./funcionario";

export interface FolhaPagamento {
    folhaPagamentoId?: number;
    funcionario?: Funcionario;
    funcionarioId: number;
    quantidadeHorasTrabalhadas: number;
    valorHora: number;
    mes: number;
    ano: number;
}

File: API-funcionario/Models/FolhaPagamento.cs
using System;

namespace API.Models
{
    public class FolhaPagamento
    {
        public int FolhaPagamentoId { get; set; }
        public Funcionario Funcionario { get; set; }
        public int FuncionarioId { get; set; }
        public int QuantidadeHorasTrabalhadas { get; set; }

        public decimal ValorHora { get; set; }

        public int Mes { get; set; }

        public int Ano { get; set; }
    }
}
